test/e2e/ownerreference_test.go

```diff
@@ -177,7 +177,7 @@ var (
 
 // cleanupVSphereObjects deletes the Secret, VSphereClusterIdentity, and VSphereDeploymentZone created for this test.
 // The VSphereFailureDomain, and the Secret for the VSphereClusterIdentity should be deleted as a result of the above.
-func cleanupVSphereObjects(ctx context.Context, bootstrapClusterProxy framework.ClusterProxy) bool {
```

Comment:
Removed the boolean return value since linter was complaining, also I do not see any use.